src/routes/safe/container/index.tsx

```diff
@@ -48,7 +48,7 @@ const Container = (): React.ReactElement => {
     return () => {
       clearTimeout(failedTimeout)
     }
-  }, [isSafeLoaded])
+  }, [isSafeLoaded, address])
```

Comment:
`address` can be removed from the dependency array since `isSafeLoaded` will update on every rerender.